config-templates/ridge/ridge-csi-driver/1.0.0/custom-parameters.json

```diff
@@ -25,6 +25,13 @@
     "required": "false",
     "category": "config"
   },
+  {
+    "description": "Ridge CSI image",
+    "displayname": "Ridge CSI image",
+    "name": "ridge-csi-image",
+    "required": "true",
```

Comment:
Since you mentioned that you need this coz you want to test this in your Dev. What we can do here is include a `"default": "Prod-image"` and set `required:false`. So now only when you want to test in dev you have to pass the dev image url.